cli/test/integration/scenarios.js

```diff
@@ -118,6 +120,53 @@ for (let scenario of scenarios) {
     })
   } // event files
 
+  const stoppedTestName = `test/scenarios/${scenario.name}/local/stopped`
+
+  if (scenario.disabled) {
+    it.skip(`${stoppedTestName} (${scenario.disabled})`, () => {})
+  } else {
+    it(stoppedTestName, async function () {
+      this.timeout(3 * 60 * 1000)
+      await Promise.delay(500)
```

Comment:
why ?

Reply:
To prevent random failures.

- [ ] Add a comment
- [ ] Fix macOS build (broken because logs are too big?)